get-started/resources.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Get Started Educational Resources"
```

Comment:
Get started educational resources?

Reply:
Changed to Educational resources to match toc title